tests/unit/WorkspaceSettingsUtilsTest.json

```diff
@@ -0,0 +1,542 @@
+                "actionName": "IOU",
+                "actorAccountID": 18634488,
+                "automatic": false,
+                "avatar": "https: //d2k5nsl2zxldvw.cloudfront.net/images/avatars/default-avatar_1.png",
+                "isAttachmentOnly": false,
+                "originalMessage": {
+                    "amount": 45600,
+                    "comment": "",
+                    "currency": "VND",
+                    "IOUTransactionID": "3690687111940510713",
+                    "IOUReportID": "6955627196303088"
+                },
+                "message": [
+                    {
+                        "deleted": "",
+                        "html": "₫456 expense",
+                        "isDeletedParentAction": false,
+                        "isEdited": false,
+                        "text": "₫456 expense",
+                        "type": "COMMENT",
+                        "whisperedTo": []
+                    }
+                ],
+                "person": [
+                    {
+                        "style": "strong",
+                        "text": "adasdasd",
+                        "type": "TEXT"
+                    }
+                ],
+                "reportActionID": "1783566081350093529",
+                "shouldShow": true,
+                "created": "2024-11-05 11: 20: 22.065",
+                "childReportID": "7900715127836904",
+                "lastModified": "2024-11-05 11: 20: 22.065",
+                "childReportNotificationPreference": "hidden",
+                "childType": "chat",
+                "reportActionTimestamp": 1730805622065,
+                "sequenceNumber": 2,
+                "timestamp": 1730805622,
+                "whisperedToAccountIDs": []
+            }
+        },
+        "reportActions_4286515777714555": {
+            "1995312838979534584": {
+                "reportActionID": "1995312838979534584",
+                "actionName": "CREATED",
+                "actorAccountID": 18634488,
+                "message": [
+                    {
+                        "type": "TEXT",
+                        "style": "strong",
+                        "text": "ajsdjajdjadjajsjajdsj123@gmail.com"
+                    },
+                    {
+                        "type": "TEXT",
+                        "style": "normal",
+                        "text": " created this report"
+                    }
+                ],
+                "person": [
+                    {
+                        "type": "TEXT",
+                        "style": "strong",
+                        "text": "adasdasd"
+                    }
+                ],
+                "automatic": false,
+                "avatar": "https: //d2k5nsl2zxldvw.cloudfront.net/images/avatars/default-avatar_1.png",
+                "created": "2024-11-05 11: 19: 18.288",
+                "shouldShow": true
+            }
+        },
+        "reportActions_7900715127836904": {
```

Comment:
Is all of this data necessary, or can you please pair it down to only the necessary data for the test?

Reply:
I removed the unnecessary data